### ISSUE TITLE
License type GPL-3.0-or-later throws warning

### ISSUE THREAD

**Issue by mmarseglia** [voxpupuli/metadata-json-lint#101]
Using metadata-json-lint (2.1.0)

Set the following in metadata.json:
```
  "license": "GPL-3.0-or-later",
```

Run tests and get this warning message:
```
(WARN) license: License identifier GPL-3.0-or-later is not in the SPDX list: http://spdx.org/licenses/
Warnings found in metadata.json
```

The expected behavior is no warnings and the test to pass.  The license is valid according to https://spdx.org/licenses/.

**Comment by rnelson0** [comment]
This appears to be related to https://github.com/domcleal/spdx-licenses/issues/1, @domcleal can you verify?

**Comment by domcleal** [comment]
It was just an outdated licence list, as the new `-or-later` IDs came in with SPDX 3.0. The gem was still on 2.4, so I've pushed a new version with the updated list. It should be fixed now if you update to `spdx-licences-1.2.0` :slightly_smiling_face: 

(The linked issue is for more complex combinations of multiple licences, which is rare in practice, at least for Puppet modules.)

**Comment by domcleal** [comment]
Please re-open if the gem update doesn't fix the issue for you.
